src/solvers/dg/3d/dg.jl

```diff
@@ -60,6 +60,9 @@ mutable struct Dg3D{Eqn<:AbstractEquation, NVARS, POLYDEG,
   amr_alpha_min::Float64
   amr_alpha_smooth::Bool
 
+  pp_limiter_apply::Bool
+  pp_limiter_threshold::Float64
```

Comment:
```suggestion
  positivity_preserving_limiter_apply::Bool
  positivity_preserving_limiter_threshold::Float64
```
I *really* like descriptive names, and `pp` is at least ambiguous in that it could mean something starting with `pressure...`

Reply:
ok